### PR TITLE
Feature/km/a11y enhancements

### PR DESCRIPTION
**Pull request by resource11**
Adds accessibility enhancements to unplatform including:
Color contrast 
Screen reader navigation
Keyboard navigation
Skip links
Focus trap management
Form validation on school config page

### CODE REVIEW

File: ui/src/routes/Lessons/Lessons.js [hunk 1]
```diff
@@ -10,6 +12,23 @@ import '../../styles/buttons.css'
 let backgroundImage = require('../../assets/clix-i2c-flowers.svg')
 
 class Lessons extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      skiplinks: [{
```
Comment (cjshawMIT) [inline ui/src/routes/Lessons/Lessons.js]: Do you need to define `skipLinks` here? Looks like it's not used, and I thought it was already defined in the state of the new `SkipLinks` component?

File: ui/src/routes/Subjects/Subjects.js [hunk 2]
```diff
@@ -1,14 +1,33 @@
...
 
 import '../../styles/components/c-breadcrumbs.css'
 
 let backgroundImage = require('../../assets/clix-i2c-flowers.svg')
 
 class Subjects extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      skiplinks: [{
```
Comment (cjshawMIT) [inline ui/src/routes/Subjects/Subjects.js]: Do you need to define `skipLinks` here? I think it's already defined in `PageFocusSection`...

File: ui/src/routes/Tool/Tool.js [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
 import '../../styles/components/c-modal.css'
 
-class Subjects extends Component {
+class Tool extends Component {
   constructor (props) {
     super(props)
     this.state = {
-      showModal: false
+      showModal: false,
+      skiplinks: [{
```
Comment (cjshawMIT) [inline ui/src/routes/Tool/Tool.js]: Do you need to define skipLinks here? I think it's already defined in PageFocusSection...

File: ui/src/routes/Tools/Tools.js [hunk 4]
```diff
@@ -1,15 +1,34 @@
...
 import '../../styles/buttons.css'
 
 let backgroundImage = require('../../assets/clix-i2c-flowers.svg')
 
-class Subjects extends Component {
+class Tools extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      skiplinks: [{
```
Comment (cjshawMIT) [inline ui/src/routes/Tools/Tools.js]: Do you need to define `skipLinks` here? I think it's already defined in `PageFocusSection`...

File: ui/src/routes/Units/Units.js [hunk 5]
```diff
@@ -10,6 +12,23 @@ import '../../styles/buttons.css'
 let backgroundImage = require('../../assets/clix-i2c-flowers.svg')
 
 class Units extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      skiplinks: [{
```
Comment (cjshawMIT) [inline ui/src/routes/Units/Units.js]: Do you need to define `skipLinks` here? I think it's already defined in `PageFocusSection`...

Comment (resource11) [reply ui/src/routes/Units/Units.js]: Ah, yes. I'll pull these extraneous skiplink state definitions out. Also found stray prop validation warning. Thanks for the catch!

**Pull request merged**